Makefile

```diff
@@ -24,6 +24,7 @@ DEBUG_SYMBOL ?= ON
 MODMESH_PROFILE ?= OFF
 BUILD_METAL ?= OFF
 BUILD_QT ?= ON
+BUILD_CUDA ?= OFF
```

Comment:
Yes, let's make it off by default.  Perhaps the variable `BUILD_CUDA` can be moved to be adjacent to `BUILD_METAL` like:

```make
BUILD_CUDA ?= OFF
BUILD_METAL ?= OFF
BUILD_QT ?= ON
```